doc/network.tex

```diff
@@ -135,107 +138,105 @@
 
-Shelley is a first fully decentralised release of Ouroboros protocol.
+Shelley is a first fully decentralised release of Cardano system, implementing the Ouroboros protocol.
 
 This Chapter contains an overview of the content and scope of this document and aspects that
 are being discussed.
 
+\wip{
 \subparagraph{Software assurance}
-\wip{Software assurance}
+Software assurance
+}
 
 \subparagraph{Layered Protocols}
 Traditionally network protocols are presented as a stack of layers where
 upper layers build on services provided by lower layers and lower layers
-are agnostic of the implementation of the upper layers.
+are independent of the implementation of the upper layers.
 This concept of layers is misleading when discussing Ouroboros.
-It is {\em not} the case that the consensus (layer) is build on top of the network (layer).
+For example, it is {\em not} the case that the consensus (layer) is built on top of the network (layer).
 It is more appropriate to talk about a network component than a network layer.
-The network component provides services to the consensus component and vice versa.
-Both components rely on each other.
-The network component uses the consensus component validate
-information that it distributes to other nodes.
-This is essential to guard against certain kinds of DoS attacks.
+The network component provides services to the consensus component and vice versa;
+both components rely on each other.
+The network component uses the consensus component to validate
+information that it distributes to other nodes, which
+is essential to guard against certain kinds of DoS attacks.
 Existing peer-to-peer systems focus on a slightly different problem domain.
 For example, they do not consider the information validation issue
-or have worries \todo{(Neil : I could not read the word)}, such as 'eclipse' that to not
+or are concerned with issues such as 'eclipse attacks' that to not
 apply to the Ouroboros family of protocols.
 
 \missingfigure[figwidth=6cm]{Testing a long text string}
 
 \subparagraph{Performance of Peer to Peer Networks}
-In computer science, Byzantine Fault Tolerance is a property which states
-that a distributed algorithm works for the honest participants of the algorithm
-under the assumption that a certain percentage of the participants is indeed honest.
+In computer science, Byzantine Fault Tolerance is a property of a distributed algorithm, which states
+that it works for the honest participants
+under the assumption that a certain proportion of the participants are indeed honest.
 A similar, but more informal property applies to peer to peer networks as well.
 
-A peer to peer network, is a system which provides a service to its participants while at the same
-time the participants of the network provide a service to the network.
+A peer to peer network is a system that provides a service to its participants while at the same
```

Comment:
Again, why are we trying to say that this is a P2P system? it isn't - we are not trying to provide a generic framework for all possible P2P actions, just support for the Data Diffusion to support the distributed execution of a concensus and ledger framework that is Ouroboros

Reply:
Very good point.. in the section I want to argue that there is a kind of recursion/ knot tying/
when discussing the performance of the network. which is true in general for P2P and
for the special case...
i would say the Ourobors fits the definition of P2P so I used P2P as synonym for Ourobros here.
If it is better to avoid the term P2P I'll think of something else